central/sensor/service/pipeline/nodeindex/pipeline.go

```diff
@@ -47,24 +50,32 @@ type pipelineImpl struct {
 	riskManager   riskManager.Manager
 }
 
-func (p pipelineImpl) OnFinish(_ string) {
+func (p *pipelineImpl) OnFinish(_ string) {
```

Comment:
I think it's ok to keep it as a value since all its contents are small (all pointers). Unless every other pipeline uses pointer receivers and this is for consistency

Reply:
I changed this due to IDE warning that I don't remember now (but I can check). I think, that the bad thing is mixing `func (p *pipelineImpl) name` with `func (p pipelineImpl) name`, the other two variants are fine as long as they are consistent.

Reply:
I checked and all other pipelines use the variant with `*`. The current version in this PR is then consistent with the rest.